ppdet/utils/checkpoint.py

```diff
@@ -113,6 +116,17 @@ def load_params(exe, prog, path, ignore_params=[]):
     logger.info('Loading parameters from {}...'.format(path))
 
     ignore_list = None
+
+    if finetune_match:
+        all_var_shape = {}
+        for block in prog.blocks:
+            for param in block.all_parameters():
+                all_var_shape[param.name] = param.shape
+        state = _load_state(path)
```

Comment:
please note this only works for single file check points, i.e., `some_model.pdparams`.

Reply:
after paddle 1.7, load_program_state supports the parameters from  [save_params, save_persistables, save_vars ]

Reply:
ah, alright, I must be confusing it with some other issue